apis/server/image_bridge.go

```diff
@@ -153,10 +152,12 @@ func (s *Server) removeImage(ctx context.Context, rw http.ResponseWriter, req *h
 	}
 
 	if err := s.ImageMgr.RemoveImage(ctx, name, isForce); err != nil {
+		metrics.ImageActionsCounter.WithLabelValues(label).Inc()
```

Comment:
if the image was not found, I think it will return err in `s.ImageMgr.GetImage(ctx, name)`, so we can judge the err here and add the counter.